src/main/java/eu/dissco/backend/client/AnnotationClient.java

```diff
@@ -13,7 +14,10 @@
 public interface AnnotationClient {
 
   @PostMapping(value = "")
-  JsonNode postAnnotation(@RequestBody AnnotationEvent event);
+  JsonNode postAnnotation(@RequestBody Annotation annotation);
+
+  @PatchMapping(value = "")
```

Comment:
As indicated in the processor PR, would be good to put to id in the url as the delete